forge-gui/res/cardsfolder/upcoming/omenport_vigilante.txt

```diff
@@ -0,0 +1,6 @@
+Name:Omenport Vigilante
+ManaCost:1 W
+Types:Creature Human Mercenary
+PT:2/2
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Double Strike | CheckSVar$ Count$CommittedCrimeThisTurn.1.0 | SVarCompare$ EQ1 | Description$ CARDNAME has double strike as long as you've committed a crime this turn. (Targeting opponents, anything they control, and/or cards in their graveyards is a crime.)
```

Comment:
bad SVarCompare... if I commit two crimes this loses double strike
just remove it as default is GE1

Reply:
On my side, the Omenport Vigilante doesn't lose double strike if I commit two crimes in a row. I verified this before uploading, but here's a screenshot for convenience. Also, if I'm understanding the logic correctly on the `CheckSVar$` based on its application on Oko, the Ringleader, it shouldn't be outputting anything other than '1' if a crime is committed.
![image](https://github.com/Card-Forge/forge/assets/45150760/653acbbb-070e-485e-9aa1-9c3afd53ac56)

Reply:
It's fair though that, with `GE1` being the default, the `SVarCompare` can be removed: worked fine after doing so.